_attributes/common-attributes.adoc

```diff
@@ -37,11 +37,33 @@
 :ocp: OpenShift Container Platform
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:ocp-supported-version: 4.15
-:rhacs-version: 4.4
-:quay-version: 3.11
+// These variable are the current product release versions for the OPP products
+:ocp-supported-version: 4.16
+:rhacs-version: 4.5
+:quay-version: 3.12
 :rhacm-version: 2.10
```

Comment:
We will have an ACM 2.11 release that will be available not long after OCP 4.16 is available.  That is the one that should be listed here.